### PR TITLE
GH-777 added daily grouping

### PR DESCRIPTION
**Pull request by DmitryBespalov**

### CODE REVIEW

File: SafeUIKit/SafeAppUI/MainFlow/MainScreen/TransactionView/TransactionsGroupHeaderView.swift [hunk 1]
```diff
@@ -9,13 +9,21 @@ import SafeUIKit
 class TransactionsGroupHeaderView: BackgroundHeaderFooterView {
 
     internal enum Strings {
-        static let pending = LocalizedString("pending_captalized",
-                                             comment: "Pending transactions group header")
+        static let pending = LocalizedString("pending_captalized", comment: "Pending transactions group header")
```
Comment (sche) [inline SafeUIKit/SafeAppUI/MainFlow/MainScreen/TransactionView/TransactionsGroupHeaderView.swift]: We decided not to use _capitalized in names (there is a typo by the way). Please use "pending" and make capitalization in code.

Comment (DmitryBespalov) [reply SafeUIKit/SafeAppUI/MainFlow/MainScreen/TransactionView/TransactionsGroupHeaderView.swift]: ok, this is a leftover from previous code

**Review by sche — CHANGES_REQUESTED** [review]
please update localization

File: SafeUIKit/SafeAppUI/MainFlow/MainScreen/TransactionView/TransactionsGroupHeaderView.swift (hunk 1 above)
Comment (DmitryBespalov) [reply SafeUIKit/SafeAppUI/MainFlow/MainScreen/TransactionView/TransactionsGroupHeaderView.swift]: btw, should it be capitalized or localizedCapitalized?

Comment (sche) [reply SafeUIKit/SafeAppUI/MainFlow/MainScreen/TransactionView/TransactionsGroupHeaderView.swift]: Never used localizedCapitalized. We use capitalized everywhere, so I would continue doing so.

**Review by sche — APPROVED** [review]

**Pull request merged**